src/main/kotlin/com/dumbdogdiner/stickysurvival/listener/GameTriggersListener.kt

```diff
@@ -0,0 +1,160 @@
+package com.dumbdogdiner.stickysurvival.listener
+
+import com.dumbdogdiner.stickysurvival.StickySurvival
+import com.dumbdogdiner.stickysurvival.event.TributeAddEvent
+import com.dumbdogdiner.stickysurvival.event.TributeRemoveEvent
+import com.dumbdogdiner.stickysurvival.util.messages
+import com.dumbdogdiner.stickysurvival.util.safeFormat
+import com.sk89q.worldedit.LocalSession
+import com.sk89q.worldedit.WorldEdit
+import com.sk89q.worldedit.bukkit.BukkitAdapter
+import com.sk89q.worldedit.command.tool.SelectionWand
+import com.sk89q.worldedit.world.item.ItemType
+import com.thevoxelbox.voxelsniper.VoxelProfileManager
+import org.bukkit.Bukkit
+import org.bukkit.Material
+import org.bukkit.entity.Player
+import org.bukkit.event.EventHandler
+import org.bukkit.event.Listener
+import org.bukkit.inventory.ItemStack
+
+/**
+ * GameTriggersListener
+ *
+ * Internal event listener for hooking into other plugins
+ * (eg. to disable external plugin functionality during games)
+ */
+object GameTriggersListener : Listener {
+
+    // Simple class for easier debug logging
+    private class GTLLogger(val topic: String) {
+        fun log(msg: String) {
+            StickySurvival.instance.logger.info("[GameTriggers $topic]: $msg")
+        }
+    }
+
+    //region Utils (VoxelSniper, WorldEdit)
+
+    private fun getVoxelProfileManager(): VoxelProfileManager? {
+        // Create a instance of GTLLogger for logging
+        val logger = GTLLogger("getVoxelProfileManager")
+
+        if (Bukkit.getServer().pluginManager.isPluginEnabled("VoxelSniper")) {
+            logger.log("plugin enabled!")
+            // Make sure the VoxelSniper plugin is loaded before trying to get VoxelProfileManager
+            val voxelProfileManager = VoxelProfileManager.getInstance()
+
+            logger.log("is null: ${(voxelProfileManager == null)}")
+
+            if (voxelProfileManager != null) {
```

Comment:
IntelliJ says "Return can be lifted out of 'if'" (returning the if expression instead of putting a return at the end of both branches) Up to you if you want to do that, I usually just do it so there's no underline on my code